ext/search_kit/ang/crmSearchDisplay/Pager.html

```diff
@@ -21,10 +21,10 @@
   </div>
   <div>
     <div class="form-inline text-right" ng-if="$ctrl.settings.pager.expose_limit && (!$ctrl.settings.pager.hide_single || $ctrl.rowCount > $ctrl.limit)">
-      <label for="crm-search-results-page-size" >
+      <label for="{{ $ctrl.tabId ? 'crm-search-results-page-size-' + $ctrl.tabId : 'crm-search-results-page-size' }}" >
         {{:: ts('Page Size') }}
       </label>
-      <input class="form-control" id="crm-search-results-page-size" type="number" ng-model="$ctrl.limit" min="10" step="10">
+      <input class="form-control" id="{{ $ctrl.tabId ? 'crm-search-results-page-size-' + $ctrl.tabId : 'crm-search-results-page-size' }}" type="number" ng-model="$ctrl.limit" min="10" step="10">
```

Comment:
@seamuslee001 this all looks good but the inline conditionals seem needlessly complex. Even if `$ctrl.tabId` is undefined it's still fine to concatenate it.